channeldb/codec.go

```diff
@@ -147,6 +147,9 @@ func WriteElement(w io.Writer, element interface{}) error {
 	case *wire.MsgTx:
 		return e.Serialize(w)
 
+	case wire.MsgTx:
```

Comment:
Nice catch, no longer needed anymore! Had this in place when I was experimenting with another way of inserting channel shells into the database (would write a blank tx instead of having the distinct channel status).